certified-connectors/Cloudmersive Currency/Readme.md

```diff
@@ -0,0 +1,36 @@
+
+## Cloudmersive Video and Media
```

Comment:
@Cloudmersive The title of your connector in "Cloudmersive Currency", It is a typo?

Reply:
@sriyen-msft Thanks for the feedback, corrected this in latest commit

Reply:
@Cloudmersive It looks like the readme that's in `Cloudmersive Currency` still says 'Cloudmersive Video and Media', but now the readme in `Cloudmersive Video and Media` says 'Cloudmersive Currency'

Reply:
@dougelliott @sriyen-msft Thanks for the feedback.  New corrections were added to address this.

Reply:
Thanks!